flink-connectors/flink-connector-hbase-1.4/src/test/java/org/apache/flink/connector/hbase1/HBaseDynamicTableFactoryTest.java

```diff
@@ -141,6 +143,32 @@ public void testTableSourceFactory() {
                 hbaseSchema.getQualifierDataTypes("f4"));
     }
 
+    @Test
+    public void testCacheOptions() {
```

Comment:
```suggestion
    public void testLookupOptions() {
```
`lookup.max-retries` is not related to cachiung.